Automattic/woocommerce-payments
# Add a card that has the element matching screenshots

Fixes https://github.com/Automattic/woocommerce-payments/issues/2174

#### Changes proposed in this Pull Request
- use `WCPaySettingsContext` context to get the UPE feature flag
- Display banner if feature flag is true, hide otherwise.

#### Follow-up
- ~Clicking "Get started" links to the onboarding wizard will be done in a follow up https://github.com/Automattic/woocommerce-payments/issues/2323~ (Changes are now included in this PR)





#### Testing instructions
1. Enable `_wcpay_feature_upe_settings_preview` and disable `_wcpay_feature_upe`  feature flag. Confirm seeing the banner:
![image](https://user-images.githubusercontent.com/572862/125879390-bf738c5c-79c1-44ee-8a0c-74cbe63743eb.png)

2. Click "Enable in your store". You should be redirected to `wp-admin/admin.php?page=wc-admin&task=woocommerce-payments--additional-payment-methods` 
![image](https://user-images.githubusercontent.com/572862/125879264-28dbe8f8-d0b9-4084-9244-4dc14320ca71.png)

3. Check database `SELECT * FROM `wp_options` where option_name like '%_wcpay_feature_upe%'`. You should see there is an entry and the value is set to `1`. 
4. Disable `_wcpay_feature_upe_settings_preview` feature flag. Confirm banner is hidden:
![image](https://user-images.githubusercontent.com/572862/123486264-ecec0b80-d5c8-11eb-9831-b81ec32783fc.png)



*

-------------------
